### DISCUSSION TITLE
Recommend Shadcn UI with NextjS CLI

### DISCUSSION THREAD

**Discussion by alamenai** [vercel/next.js#70884] [Ideas]
### Goals

1. Integrating Shadcn with NextJS configuration once time.

### Non-Goals

1.
2.
3.

### Background

When I create a new project from scratch each time, I have to install Shadcn each time using Shadcn CLI, so I asked myself if NextJS recommends TailwindCSS, why not also Shadcn?

### Proposal

Updating the CLI by adding a new command that runs Shadcn internally.

**Comment by suprunchuk** [comment]
It seems to me that it is not so difficult to enter [one command](https://ui.shadcn.com/docs/installation/next) `npx shadcn@latest init` yet
![image](https://github.com/user-attachments/assets/44dbcd64-424e-4449-a1d6-b01f7d9e9986)
